toolchain/check/testdata/builtins/int/make_type_signed.carbon

```diff
@@ -78,7 +80,7 @@ library "[[@TEST_NAME]]";
 
 import library "types";
 
-fn Negate(n: i32) -> i32 = "int.snegate";
+fn Negate(a: IntLiteral()) -> IntLiteral() = "int.snegate";
```

Comment:
As before, can you please split out test changes?

Reply:
> As before, can you please split out test changes?

Done, this PR no longer contains test changes.